### ISSUE TITLE
Support for getting Badges

### ISSUE THREAD

**Issue by msoucy** [andymeneely/game_icons#11]
The game-icons "Studio" has support for Badges, a set of small additions to the regular icons. It would be nice to have a way to get those, whether that be individually and manually placed or as part of a bigger SVG. Some of the badges don't really have an analogous "base" image

**Comment by andymeneely** [comment]
Interesting. I'll take a look.

**Comment by andymeneely** [comment]
That's a cool feature! I can try to work this in, although most of the time I would do that in Squib anyway. At the very least I should have a "badge" method to just get the base image. If you just want the images, they are here: https://github.com/game-icons/icons/tree/master/badges

**Comment by msoucy** [comment]
I agree that Squib could probably handle many (most?) of the use cases, though I think it would be difficult to do a badge in an svg embedded into text. (It remains to be seen whether they're actually be clearly visible in that case of course)

Annoyingly, the link you sent is missing the numeric badges, which was sort of the entire point of what I wanted the badges for... I can try updating the gem locally and see if I can load 'badges/foo' later today as an ordinary svg.
